DuoComposeSampleApps/ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.device.display.samples.listdetail
```

Comment:
I think you removed the dummy test from other composable samples, so we can delete this as well.